src/etl/adapter/yd_parser.py

```
@@ -0,0 +1,122 @@
+        """
+        public_key: ссылка на Яндекс.Диск
+        """
+        all_messages = []
+        
+        # 1. Создаем временную директорию для работы
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            zip_path = os.path.join(tmp_dir, "download.zip")
+            extract_path = os.path.join(tmp_dir, "extracted")
+
+            # 2. Получаем прямую ссылку на скачивание через API Яндекса
+            response = requests.get(f"{self.base_url}{public_key}")
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)